javatests/build/bazel/tests/integration/BUILD

```diff
@@ -31,8 +31,32 @@ bazel_java_integration_test(
     runtime_deps = [":BazelIntegrationTestNoSourcesSrc"],
 )
 
+genrule(
+    name = "test_archive",
+    outs = ["ext.zip"],
+    cmd = """
+# From https://github.com/bazelbuild/bazel/blob/88374614b62b17587453137814d264fdaf7bd70e/src/test/shell/bazel/external_integration_test.sh
```

Comment:
how does this work if you can't download it on build time? I think it downloads it on build time and I'm not sure I understand why we need to

Reply:
What do you mean?

Reply:
🤦‍♂️ missed that this is a comment...
The purpose of this genrule is an archive just for **our** own test? If so wdyt about renaming the output to something like `ext.zip`->`externalDependenciesArchiveForInternalTest.zip`? and maybe the name of the genrule to something else since it loads the name of the external repository and at least for me it wasn't clear how they interact